src/Auth.php

```diff
@@ -296,12 +296,12 @@ public function reject($message = null)
         print '  <th class="form-label_left">';
         print 'Use<span class="accesskey">r</span>name:</th>' . "\n";
         print '  <td class="form-input">';
-        print '<input size="20" name="PECL_USER" accesskey="r" /></td>' . "\n";
+        print '<input size="20" name="PECL_USER" accesskey="r" autocomplete="username" required="" /></td>' . "\n";
```

Comment:
Does `required=""` actually require input? Shouldn't that be `required="required"` instead (or just `required`)?